### ISSUE TITLE
Firefox 38.8.0 - print progress dialog does not close

### ISSUE THREAD

**Issue by lerdmann** [bitwiseworks/mozilla-os2#180]
If you print, the progress dialog will not close even if the job terminates successfully. Hitting the "Cancel" button also shows no effect. You have to close the progress dialog via the "cross" button.

**Comment by herwigb** [comment]
I can confirm this one.

**Comment by SilvanScherrer** [comment]
Known issue and will be worked on when we work on printing  #170 and #171

**Comment by dmik** [comment]
BTW, it's not reproducible with the current test RPM of Firefox. And Silvan confirms the same on one of his machines.

**Comment by dspiatkowski** [comment]
Hi Dimitriy,

On Thu, 28 Sep 2017 13:12:55 +0000 (UTC), Dmitriy Kuminov wrote:

>BTW, it's not reproducible with the current test RPM of Firefox. And Silvan confirms the same 
on one of his machines.

I am running the lxlited patch9 release of FF right now...is there a way for me to pull the RPM 
version and install separately/manually as opposed to having it auto-installed? I have not been 
able to find it by manually traversing the web/ftp-server structure...

The reason I am asking is that for an application like FF I want there to be a specific directory 
it goes into, as opposed to the /usr/bin... stuff.

Thanks,
-Dariusz

**Comment by dspiatkowski** [comment]
Hi Dmitriy,

On Fri, 29 Sep 2017 05:35:40 -0700, Dmitriy Kuminov wrote:

>@dspiatkowski You may unpack the contents of `/usr/lib/firefox-XYZ` from RPM to any dir 
 Çö it should just work like that (but note that while we try to keep such a custom install 
working, this isn't guaranteed in the future and generally not supported).


No, I got that...no problem there. My issue was that I could not actually find the firefox RPM 
package anywhere out on the netlabs servers.

So for example, a place like http://rpm.netlabs.org/release/00/pentium4/i686/ is where I would 
normally find the PRM packages, but firefox is not there, yet I can see the package itself in 
ANPM. Then it hit me...ohhh...I expected the package to be in the netlabs-exp repo, but it's 
actually being published to arcanoae-exp, found it!!!

OK, so that release is dated: 31-Aug-2017 10:59, so how often are the changes you are 
making being rolled into the test releases? 

I would love to test with the latest release so that similar issues are not being re-reported 
multiple times...take for example the Google maps case...I think on the OS2World forum we got 
into a whole mix of people seeing different results across numerous versions.

Thanks again,
- Dariusz
